src/Xz/XzBinaryLocator.php

```diff
@@ -19,7 +19,13 @@ public function __construct(?string $path)
     public function getXzBinary(): string
     {
         if ($this->path === null) {
-            exec('which xz', $output, $return);
+            if (stristr(PHP_OS, 'LINUX')) {
+                exec( 'which xz', $output, $return );
+            } elseif (stristr(PHP_OS, 'WIN')) {
+                exec( 'where xz', $output, $return );
+            } else {
+                throw new QrPaymentException("not supported OS");
```

Comment:
I would like more descriptive exception message. Including mention of the setXzBinary() method to help guide people on how to solve the error.

Reply:
added